### ISSUE TITLE
SRIOV VirtualMachineInstance with sriov plugin interface [test_id:3956]should connect to another machine with sriov interface over IPv4

### ISSUE THREAD

**Issue by arnongilboa** [kubevirt/kubevirt#3774]
/kind bug
/triage build-officer

**What happened**:

```
[Fail] SRIOV VirtualMachineInstance with sriov plugin interface [It] [test_id:3956]should connect to another machine with sriov interface over IPv4 
/root/go/src/kubevirt.io/kubevirt/tests/vmi_multus_test.go:896
```

https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3765/pull-kubevirt-e2e-kind-1.17-sriov/1280125066415181825

**Comment by brybacki** [comment]
and again:
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3755/pull-kubevirt-e2e-kind-1.17-sriov/1280448320728207362

**Comment by alonSadan** [comment]
hopefully #4001 will solve it.
/assign @alonSadan

**Comment by ormergi** [comment]
More details at https://github.com/kubevirt/kubevirt/issues/4432

**Comment by oshoval** [comment]
> More details at #4432

4432 is not related, error here is eth1 not found, on 4432 its the ping that doesnt work (when i logged in the machine i saw the device is there but ip was gone, reasons on 4432)

**Comment by ormergi** [comment]
This issue is user report for a flaky test.
This test is flaky for two reasons, one is resolved and described by https://github.com/kubevirt/kubevirt/pull/4001 
And the second is describe by https://github.com/kubevirt/kubevirt/issues/4432

**Comment by oshoval** [comment]
Fixed by #4448 by @qinqon 
/close
